backend/init_data/skills/browser/_base.py

```
@@ -0,0 +1,375 @@
+                from playwright.async_api import async_playwright
+
+                logger.info("[BrowserSessionManager] Playwright is already installed")
+            except ImportError:
+                logger.info(
+                    "[BrowserSessionManager] Playwright not found, installing..."
+                )
+                # Install playwright using pip
+                import subprocess
+
+                result = subprocess.run(
+                    ["pip", "install", "playwright"],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)